pkg/grpc/universalapi/api_actors.go

```diff
@@ -47,3 +53,51 @@ func (a *UniversalAPI) WaitForActorsReady(ctx context.Context) {
 	case <-a.actorsReadyCh:
 	}
 }
+
+// This function makes sure that the actor subsystem is ready.
+func (a *UniversalAPI) actorReadinessCheck(ctx context.Context) error {
+	a.WaitForActorsReady(ctx)
+
+	if a.Actors == nil {
+		a.Logger.Debug(messages.ErrActorRuntimeNotFound)
+		return messages.ErrActorRuntimeNotFound
+	}
+
+	return nil
+}
+
+func (a *UniversalAPI) DeleteActorState(ctx context.Context, in *runtimev1pb.DeleteActorStateRequest) (*emptypb.Empty, error) {
+	if err := a.actorReadinessCheck(ctx); err != nil {
+		return &emptypb.Empty{}, err
+	}
+
+	actorType := in.ActorType
```

Comment:
Nit: No need to copy these in a local variable, since they are not changed. Just use `in.ActorType` in all cases below. (Same for `in.ActorId` and `in.Key`)